### PR TITLE
To review

### PR DESCRIPTION
**Pull request by ragog**

### CODE REVIEW

**Comment by KevinMarkVI** [comment]
Looks good to me. @ndmanvar ?

File: src/test/java/com/yourcompany/SampleSauceTest.java [hunk 1]
```diff
@@ -17,7 +19,7 @@
 import org.openqa.selenium.OutputType;
 
 
-@TestObject(testObjectApiKey = "INSERT_API_KEY_HERE", testObjectSuiteId = INSERT_SUITE_NUMBER_HERE)
+@TestObject(testObjectApiKey = "INSERT_API_KEY_HERE", testObjectSuiteId = 12345)
```
Comment (ndmanvar) [inline src/test/java/com/yourcompany/SampleSauceTest.java:22]: @ragog I don't think we should hard-code the testId?
Let's remove that and use `"INSERT_SUITE_NUMBER_HERE"` or an environment variable?

Happy to discuss.

**Comment by ndmanvar** [comment]
Just one small change regarding SUITE. Other than that, good to me! 👍

**Review by ndmanvar — CHANGES_REQUESTED** [review]

File: src/test/java/com/yourcompany/SampleSauceTest.java (hunk 1 above)
Comment (ragog) [reply src/test/java/com/yourcompany/SampleSauceTest.java:22]: @ndmanvar the testObjectSuiteId is automatically overridden in this case (we are listening for the environment variable within our library). Putting a random value there at least allows us to override it without running into an error when we try to compile. You can only insert constant values in annotations, too, so we are kind of limited in how we approach this.

**Pull request merged**